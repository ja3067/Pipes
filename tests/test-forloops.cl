l = [7, 9, 5]

x = 3.0

for i in l:
	a = i
	x = i
	print(a)
	print(x)
	print(i)

print(a)
print(i)
print(x)
