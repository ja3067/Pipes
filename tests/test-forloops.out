7
7
7
9
9
9
5
5
5
5
5
5
